Chocolatey.Cake.Recipe/Content/toolsettings.cake

```diff
@@ -62,6 +63,7 @@ public static class ToolSettings
         string reportUnitTool = "#tool nuget:?package=ReportUnit&version=1.2.1",
         string reSharperReportsTool = "#tool nuget:?package=ReSharperReports&version=0.2.0",
         string reSharperTools = "#tool nuget:?package=JetBrains.ReSharper.CommandLineTools&version=2017.2.0",
+        string sonarQubeTool = "#tool nuget:?package=MSBuild.SonarQube.Runner.Tool",
```

Comment:
Does this tool work on all Operating Systems?  Or do we need to gate SonarQube execution to only run on Windows? i.e. what will happen when the tagged build runs on GitHub Actions for Mac and Linux?